public/js/lib/highlight/externals.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+import {escapeHtml}  from './utils'
+
+let externals = {}
+
+externals.checkLang = function (lang) {
+  if (typeof lang !== 'string') {
+    return false
+  }
+  const languages = ['sequence', 'flow', 'graphviz', 'mermaid', 'abc']
```

Comment:
(basically we could just return this and have the surrounding code figure it out)